anewb87/overlook-hotel
# Feature/html round1

#### What's this PR do?
- Begins to build out some html and css
- Adds a number of `.scss` files for better organization
- Adds my first wireframe - very simple but pretty cool


#### Where should the reviewer start?
- A number of files have been added: `_keyframes.scss`, `customer-dashboard.scss` (nothing in this yet), `header.scss`
- Some css has been added to a number of the files in the css folder


#### How should this be manually tested?
Once all set up (clone to machine)
- npm start
- open `http://localhost:8080/`
- not much yet to see but it's coming along


#### Any background context you want to provide?
Nope

#### Questions:
Having padding/margin issues on the placeholder for the user's name. Not sure why it's going off the page like it is. Will need to find a fix.
